ethereum-types/src/hash.rs

```diff
@@ -30,7 +30,12 @@ impl_fixed_hash_serde!(H32, 4);
 #[cfg(feature = "codec")]
 impl_fixed_hash_codec!(H32, 4);
 
-construct_fixed_hash! { pub struct H64(8); }
+construct_fixed_hash! {
+	//TODO I guess we'll need this on a lot of them. Maybe I can put it in the macro?
+	// But where is the macro defined?
+	#[cfg_attr(feature = "codec", derive(scale_info::TypeInfo))]
+	pub struct H64(8);
+}
```

Comment:
I could use some help on this part.

Reply:
The macro is [here](https://github.com/paritytech/parity-common/tree/master/fixed-hash). It'd be good to not have to add a `scale-info` dependency there so what you did here looks right to me and if it doesn't work we can perhaps sort it out on the macro side?